aws/ecs/service-only-with-volume/README.md

```diff
@@ -24,6 +24,7 @@ This module is used to create an ECS service as well as task definition
 
  - `task_role_arn` - ARN for role to assign to task definition. Default: `blank`
  - `network_mode` - Networking mode for task. Default: `bridge`
+ - `execution_role_arn` - ARN for execution role that allows ECS to pull container images and secrets from SSM Parameter Store. Default: `blank`
```

Comment:
Since the execution role ARN is much broader in purpose, I pulled some text from [the docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_execution_IAM_role.html) that you might want to include.
```suggestion
 - `execution_role_arn` - ARN for execution role that allows ECS to make AWS API calls on your behalf, such as to pull container images from ECR when using Fargate or to reference secrets from SSM Parameter Store. Default: `blank`
```